### PR TITLE
Data integrity trails in DataShard

### PR DESCRIPTION
**Pull request by SammyVimes**
### Changelog category 

* Experimental feature

How it looks like: https://gist.github.com/SammyVimes/a25a4db93c9d17d950d68a32bf2b7a5b

### CODE REVIEW

File: ydb/tools/integrity_trailer/main.cpp [hunk 1]
```diff
@@ -0,0 +1,262 @@
+#include <library/cpp/json/json_value.h>
```
Comment (va-kuznecov) [inline ydb/tools/integrity_trailer/main.cpp:1]: please move this programm to another PR

Comment (SammyVimes) [reply ydb/tools/integrity_trailer/main.cpp]: Done

**Review by azevaykin — DISMISSED** [review]

**Review by azevaykin — APPROVED** [review]
